Jenkinsfile

```diff
@@ -159,12 +162,13 @@ pipeline {
                 }
             }
         }
-        // stage('Cleanup') {
-        //     steps {
-        //         script {
-        //             sh "./player.sh sparsify"
-        //         }
-        //     }
-        // }
+        stage('Cleanup') {
+             steps {
+                 agent { label 'code-tests' }
```

Comment:
This is probably the issue

Reply:
This seems to have been fixed now, and now runs successfully.

```none
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Cleanup)
[Pipeline] node
Running on code-tests in /home/jenkins/workspace/Development_PR-747
[Pipeline] {
[Pipeline] checkout
Cloning the remote Git repository
Honoring refspec on initial clone
Cloning repository https://github.com/bcgov/moh-prime.git
 > git init /home/jenkins/workspace/Development_PR-747 # timeout=10
Fetching upstream changes from https://github.com/bcgov/moh-prime.git
 > git --version # timeout=10
using GIT_ASKPASS to set credentials PR Credentials
 > git fetch --tags --force --progress -- https://github.com/bcgov/moh-prime.git +refs/pull/747/head:refs/remotes/origin/PR-747 +refs/heads/develop:refs/remotes/origin/develop # timeout=10
 > git config remote.origin.url https://github.com/bcgov/moh-prime.git # timeout=10
 > git config --add remote.origin.fetch +refs/pull/747/head:refs/remotes/origin/PR-747 # timeout=10
 > git config --add remote.origin.fetch +refs/heads/develop:refs/remotes/origin/develop # timeout=10
 > git config remote.origin.url https://github.com/bcgov/moh-prime.git # timeout=10
Pruning obsolete local branches
Fetching upstream changes from https://github.com/bcgov/moh-prime.git
using GIT_ASKPASS to set credentials PR Credentials
 > git fetch --tags --force --progress --prune -- https://github.com/bcgov/moh-prime.git +refs/pull/747/head:refs/remotes/origin/PR-747 +refs/heads/develop:refs/remotes/origin/develop # timeout=10
Merging remotes/origin/develop commit aa31ebda95b26ce103c407dffc8060f3a2da1c33 into PR head commit 9eb1dbb41cf06c7368ca8d5c6cd02f65d828b435
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 9eb1dbb41cf06c7368ca8d5c6cd02f65d828b435 # timeout=10
 > git remote # timeout=10
 > git config --get remote.origin.url # timeout=10
using GIT_ASKPASS to set credentials PR Credentials
 > git merge aa31ebda95b26ce103c407dffc8060f3a2da1c33 # timeout=10
 > git rev-parse HEAD^{commit} # timeout=10
Merge succeeded, producing 9eb1dbb41cf06c7368ca8d5c6cd02f65d828b435
Checking out Revision 9eb1dbb41cf06c7368ca8d5c6cd02f65d828b435 (PR-747)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 9eb1dbb41cf06c7368ca8d5c6cd02f65d828b435 # timeout=10
Commit message: "Merge branch 'develop' into fix/fix-zap"
[Pipeline] withEnv
[Pipeline] {
[Pipeline] sh
[Development_PR-747] Running shell script
+ ./player.sh sparsify
Deleting all non-configuration subdirectories...
```